catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/DynamicSchemaResolver.java

```diff
@@ -156,9 +160,18 @@
   protected Cache<String, byte[]> metacardTypeNameToSerialCache =
       CacheBuilder.newBuilder().maximumSize(4096).initialCapacity(64).build();
 
+  protected List<String> additionalFields = new ArrayList<>();
```

Comment:
❓ Can we not expose the internal list for testing? This makes the tests brittle. It also opens up the implementation to unexpected changes.